docs/customization.md

```diff
@@ -174,8 +174,13 @@ This content appears at the bottom of every page's main content. More info for t
 
 Any HTML added to this file will be inserted before the closing `<head>` tag. This might include additional `<meta>`, `<link>`, or `<script>` tags.
 
-#### Example
-{: .no_toc }
+Note that by default, this file has the following contents:
+
+```html
+<link rel="shortcut icon" href="{{ 'favicon.ico' | relative_url }}" type="image/x-icon">
+```
+
+#### Example: Custom Favicon
```

Comment:
The removal of `{ .no_toc }` gives this in the displayed TOC:

<img width="350" alt="image" src="https://user-images.githubusercontent.com/18308236/204876318-4a283472-1c37-4714-8d38-68bbb5969a2d.png">

To me, the pointless `a` on the example line is distracting, and it highlights a weakness of the theme: the lack of a sensible labelling for level 3 of nested `<ol>`s.

A workaround would be to restore the `{ .no_toc }` on the example heading, and change the section title to `Custom Head and Favicon`.

Reply:
Also great suggestion, I'll use your suggested section title.